libs/jwst/src/workspace/metadata/pages.rs

```diff
@@ -39,6 +42,20 @@ impl PageMeta {
 
 impl<T: ReadTxn> From<(&T, MapRef)> for PageMeta {
     fn from((trx, map): (&T, MapRef)) -> Self {
+        let sub_page_ids = match map.get(trx, "subpageIds").map(|v| v.to_json(trx)) {
+            Some(Any::Array(sub_pages)) => sub_pages
+                .iter()
+                .map(|sub_page| {
+                    if let Any::String(str) = sub_page {
+                        str.to_string()
+                    } else {
+                        panic!("Expected String in sub-page IDs array");
```

Comment:
dont panic, use TryFrom

Reply:
Is it possible to change it to what it is now?